### PR TITLE
Upgrade AWS SDK to v3

### PR DESCRIPTION
**Pull request by halkyon**

### CODE REVIEW

**Comment by stojg** [comment]
tests are broken :trollface:

**Comment by halkyon** [comment]
Tests fixed!

File: src/SilverStripe/BlowGun/Credentials/BlowGunCredentials.php [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 namespace SilverStripe\BlowGun\Credentials;
 
-use Aws\Common\Credentials\Credentials;
+use Aws\Credentials\CredentialProvider;
+use RuntimeException;
 
-class BlowGunCredentials extends Credentials
+class BlowGunCredentials
```
Comment (stojg) [inline src/SilverStripe/BlowGun/Credentials/BlowGunCredentials.php]: This hardly (and probably never was) used as a `Credential`. A quick glance seems to indicate that all the `self::` references still pointing to this class and the `CredentialProvider`. Should we rename the class to something that wouldn't cause confusion in the future?

Comment (halkyon) [reply src/SilverStripe/BlowGun/Credentials/BlowGunCredentials.php]: yeah I think it's more of a `Util` class than anything, will change it to be less confusing

Comment (halkyon) [reply src/SilverStripe/BlowGun/Credentials/BlowGunCredentials.php]: fixed!

**Comment by halkyon** [comment]
Pre-release here for testing: https://code.platform.silverstripe.com/aws/aws-projects/merge_requests/545

**Pull request merged**